oat-sa/extension-tao-qtiprint
# fix: remove doted borders for interactions

Related to:
https://oat-sa.atlassian.net/browse/BSA-344

STEPS TO TEST:

1.Create a Test containing various interactions [inlineChoice, choice, gapMatch...]
2. Create a Booklet from that test [to be able to do this in local it is necessary to have booklet extension installed, hints on debugging issues[ here ](https://oat-sa.atlassian.net/wiki/spaces/~726792781/pages/1886224743/Debug+taoBooklet#Troubleshoot)
3.  Go to booklet and generate preview.

CURRENT RESULT:
Some interactions show a dotted line around.

EXPECTED RESULT:
No dotted line is present around any interactions. 
![image](https://user-images.githubusercontent.com/60346520/199004795-a409c74a-3905-4b12-815a-71e1cb36c90b.png)

:warning: FYI => We couldn't find the way to fix the pdf download to check that borders are also no present in pdf.
As pdf is created from HTML, work around has been creating booklet preview, right clicking in preview iframe and then ` print`  to be able to see printable version.

![image](https://user-images.githubusercontent.com/60346520/199019395-a6de359f-1213-464c-a649-dcd33ba69056.png)
